internal/backend/sessions/config.go

```diff
@@ -28,6 +28,7 @@ var defaultConfig = Config{
 			ActivityScheduleToCloseTimeout: time.Hour,
 			ActivityStartToCloseTimeout:    time.Minute * 30,
 			LocalScheduleToCloseTimeout:    time.Second * 5,
+			ActivityScheduleToStartTimeout: time.Second * 5,
```

Comment:
5s is reasonable for local activities (because the worker is already running by definition), but for potentially-remote activities it means a likelihood of activity loss because we won't tolerate worker restarts.

Also note the implication for `ActivityScheduleToCloseTimeout`:

`ActivityScheduleToStartTimeout` + `ActivityStartToCloseTimeout`  = 30m5s << 1h = `ActivityScheduleToCloseTimeout`

Reply:
> 5s is reasonable for local activities (because the worker is already running by definition), but for potentially-remote activities it means a likelihood of activity loss because we won't tolerate worker restarts.

This is used only for activities on unique workers.

> 
> Also note the implication for `ActivityScheduleToCloseTimeout`:
> 
> `ActivityScheduleToStartTimeout` + `ActivityStartToCloseTimeout` = 30m5s << 1h = `ActivityScheduleToCloseTimeout`

increased `StartToClose` to 45m.